### ISSUE TITLE
ACF Repeater

### ISSUE THREAD

**Issue by Pixelatex** [VaLeXaR/wp-multilang#71]
My ACF Repeater fields:

```
'features'
-> 'feature_title'
-> 'feature_content'
```

My wpm-config.json file:

```
{
  "post_fields": {
    "features": {
      "wpm_each": {
        "feature_title": {},
        "feature_content": {}
      }
    }
  }
}
```
I also tried adding them like this:

```
{
  "post_fields": {
    "feature_title": {},
    "feature_content": {},
    "features": {}
  }
}
```

But they refuse to translate. They keep overwritting the previous values. I've seen this issue popup on the wordpress plugin forum a few times without a fix.

Can you help me figure out what's going wrong?

**Comment by VaLeXaR** [comment]
Hi. "wpm_each" using for translating multitlingual arrays in database. You have to know how  fields saved in database.

**Comment by VaLeXaR** [comment]
If it is text field type, then it translating by default.

**Comment by RuslanAsadov** [comment]
> If it is text field type, then it translating by default.

Is it possible to use a repeater?

**Comment by ahmedabbous** [comment]
@VaLeXaR thats not true , when it is text fields inside repeater it will not be translated

**Comment by ahmedabbous** [comment]
@VaLeXaR Better if you update your repo where it says compatible  with ACF Pro, because its NOT

**Comment by RuslanAsadov** [comment]
@VaLeXaR if there was support for the options page and repeater there would be a better plugin 100%!
